tests/rbd/rbd_utils.py

```
@@ -919,3 +920,24 @@ def verify_migration_commit(rbd, pool, image):
     else:
         log.error((f"Image {image} is not found in pool {pool}"))
         raise ImageNotFoundError("Image Not found.")
+
+
+def verify_migration_abort(rbd, pool, image):
+    """Verify migration commit.
```

Instead of `commit` it should be `abort` i guess, action performing is commit but method name says abort

---

right . done changes